robottelo/cli/base.py

```diff
@@ -431,7 +433,7 @@ def set_parameter(cls, options=None):
         return result
 
     @classmethod
-    def update(cls, options=None):
+    def update(cls, options=None, user=None, password=None):
```

Comment:
I'm against extending _some_ of basic commands, they'd better be all the same format. Have you checked `with_user()` decorator? The usage is like this:
```python
            Repository.with_user(
                user['login'],
                user['password']
            ).synchronize({u'id': repo['id']})
```
I think this may be applied to your tests instead of such workarounds. You may combine it with custom helpers like `update_user()` which you already implemented to be able not to include username/password for each executed command.